src/senselab/audio/tasks/text_to_speech/marstts.py

```diff
@@ -0,0 +1,150 @@
+"""This module provides the implementation of Mars5-TTS-based text-to-speech pipelines."""
+
+import time
+from typing import Dict, List, Optional, Tuple
+
+import torch
+
+from senselab.audio.data_structures.audio import Audio
+from senselab.utils.data_structures.device import DeviceType, _select_device_and_dtype
+from senselab.utils.data_structures.language import Language
+from senselab.utils.data_structures.logging import logger
+from senselab.utils.data_structures.model import TorchModel
+
+
+class Mars5TTS:
+    """A class for managing Torch-based Mars5TTS models."""
+
+    _models: Dict[str, Tuple[torch.nn.Module, type]] = {}
+
+    @classmethod
+    def _get_torch_tts_model(
+        cls,
+        model: TorchModel = TorchModel(path_or_uri="Camb-ai/mars5-tts", revision="master"),
+        language: Optional[Language] = None,
+        device: Optional[DeviceType] = None,
+    ) -> Tuple[torch.nn.Module, type]:
+        """Get or create a Torch-based Mars5TTS model.
+
+        Args:
+            model (TorchModel): The Torch model (default is "Camb-ai/mars5-tts").
+            language (Optional[Language]): The language of the text (default is None).
+                The only supported language is "en" for now.
+            device (DeviceType): The device to run the model on (default is None). Supported devices are CPU and CUDA.
+
+        Returns:
+            model: The Torch-based Mars5TTS model.
+            config_class: The configuration class used by the model.
+        """
+        if model.path_or_uri != "Camb-ai/mars5-tts" or model.revision != "master":
+            raise NotImplementedError("Only the 'Camb-ai/mars5-tts' model is supported for now.")
+        if language == Language(language_code="en"):
```

Comment:
This requires `language="en"`, which will make the function throw an error with the default setting (`language=None`)